components/browser/search/src/main/java/mozilla/components/browser/search/SearchEngineManager.kt

```diff
@@ -99,6 +99,15 @@ class SearchEngineManager(
         }
     }
 
+    /**
+     * Returns the provided default search engine if the user has no preference.
```

Comment:
documentation nit: "if the user has no preference" is not fully true: This will always provide the default even if the user has selected a different one.